src/app/@ansyn/ansyn/modules/menu-items/cases/actions/cases.actions.ts

```diff
@@ -33,31 +31,31 @@ export const CasesActionTypes = {
 
 	SAVE_CASE_AS: 'SAVE_CASE_AS',
 	SAVE_CASE_AS_SUCCESS: 'SAVE_CASE_AS_SUCCESS',
-	
-	COPY_CASE_LINK: 'COPY_CASE_LINK',
+	SAVE_SHARED_CASE_AS_MY_OWN: 'SAVE_SHARED_CASE_AS_MY_OWN',
 
-	SET_DEFAULT_CASE_QUERY_PARAMS: 'SET_DEFAULT_CASE_QUERY_PARAMS',
-	REMOVE_DEFAULT_CASE_QUERY_PARAMS: 'REMOVE_DEFAULT_CASE_QUERY_PARAMS',
-	TOGGLE_FAVORITE_OVERLAY: 'TOGGLE_FAVORITE_OVERLAY',
+	COPY_CASE_LINK: 'COPY_CASE_LINK',
 
 	LOAD_DEFAULT_CASE_IF_NO_ACTIVE_CASE: 'LOAD_DEFAULT_CASE_IF_NO_ACTIVE_CASE',
-	MANUAL_SAVE: 'MANUAL_SAVE',
-
-	SET_AUTO_SAVE: 'SET_AUTO_SAVE',
 
-	LOG_RENAME_CASE: 'LOG_RENAME_CASE'
+	RENAME_CASE: 'RENAME_CASE'
 };
 
 export type CasesActions = any;
 
+export class ShowCasesTableAction implements Action {
+	readonly type = CasesActionTypes.SHOW_CASES_TABLE;
+	constructor(public payload: boolean = false) {
+	}
+}
+
 export class LoadDefaultCaseIfNoActiveCaseAction implements Action {
 	type = CasesActionTypes.LOAD_DEFAULT_CASE_IF_NO_ACTIVE_CASE;
 }
 
 export class LoadCasesAction implements Action, ILogMessage {
 	type = CasesActionTypes.LOAD_CASES;
 
-	constructor(public payload?: ICase[]) {
+	constructor(public payload: CasesType = CasesType.MyCases, public fromBegin = false) {
```

Comment:
when user save shared case. we load his case again.
so we ask them from begin, because if not we ask them from the `offset` and don't get the new one.